sdk/schemaregistry/Azure.Data.SchemaRegistry/src/autorest.md

```diff
@@ -22,11 +49,19 @@ directive:
       "in": "header",
       "description": "Content type of the schema.",
       "required": true,
-      "type": "string"});
-     $.responses["415"]["x-ms-error-response"] = true
+      "type": "string",
+      "enum": [
+        "application/json; serialization=Avro",
```

Comment:
As you discovered, we can't do this because it would break the ToString behavior.